sdk/cmake/gcc.cmake

```diff
@@ -423,30 +431,41 @@ add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:$<IF:$<BOOL:$<TARGET_PROPERTY:WIT
 # We disable exceptions, unless said so
 add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:$<IF:$<BOOL:$<TARGET_PROPERTY:WITH_CXX_EXCEPTIONS>>,-fexceptions,-fno-exceptions>>")
 
+# G++ shipped with ROSBE uses sjlj exceptions. Tell Clang it is so
```

Comment:
this is for C++ exceptions.
And yes, CLang supports SEH, but it doesn't work right now...

Reply:
> this is for C++ exceptions

I know, I put the `--enable-sjlj-exceptions` setting into the toolchain myself :)
MSVC uses SEH for C++ exceptions, so should Clang, shouldn't it?

Reply:
I didn't push the investigation very far. Without this, C++ code has references to undefined symbols. I think this would be related to SJLJ vs DWARF exceptions unwinding, not SEH.